servicerouter.js

```diff
@@ -360,15 +366,25 @@ class ServiceRouter {
   }
 
   /**
-  * @name markSocket
-  * @summary Tags a websocket with an ID
+  * @name sendConnectMessage
+  * @summary Send a message on socket connect
   * @param {object} ws - websocket
+  * @param {number} id - connection id if any
   */
-  markSocket(ws) {
-    ws.id = Utils.shortID();
+  sendConnectMessage(ws, id) {
+    ws.id = id || `${Utils.shortID()}`;
```

Comment:
Is there a reason for the string interpolation here instead of just `id || Utils.shortID()`?